src/python_testing/TC_DeviceBasicComposition.py

```diff
@@ -166,6 +166,10 @@ def check_no_duplicates(obj: Any) -> None:
         raise ValueError(f"Value {str(obj)} contains duplicate values")
 
 
+def is_test_vendor(id) -> bool:
+    return 0xFFF1_0000 <= id <= 0xFFF4_FFFF
```

Comment:
```suggestion
def is_test_vendor(id: int) -> bool:
    vendor_id = (id >> 16) & 0xFFFF
    return 0xFFF1 <= vendor_id <= 0xFFF4
```

Reply:
There are actually functions for this already - this test just happens to predate those and wasn't updated. Look in global_attribute_ids.py

Reply:
Thanks, Cecille. I removed this function and used the checks in  global_attribute_ids.py.
I'll update the test and add checks for commands in a separate PR.